migrations/versions/0082_add_golive_template.py

```diff
@@ -0,0 +1,116 @@
+Create Date: 2017-05-10 16:06:04.070874
+
+"""
+
+# revision identifiers, used by Alembic.
+from datetime import datetime
+
+from flask import current_app
+
+from alembic import op
+import sqlalchemy as sa
+
+revision = '0082_add_go_live_template'
+down_revision = '0081_noti_status_as_enum'
+
+template_id = '618185c6-3636-49cd-b7d2-6f6f5eb3bdde'
+
+
+def upgrade():
+    template_insert = """
+        INSERT INTO templates (id, name, template_type, created_at, content, archived, service_id, subject, created_by_id, version, process_type)
+        VALUES ('{}', '{}', '{}', '{}', '{}', False, '{}', '{}', '{}', 1, '{}')
+    """
+    template_history_insert = """
+        INSERT INTO templates_history (id, name, template_type, created_at, content, archived, service_id, subject, created_by_id, version, process_type)
+        VALUES ('{}', '{}', '{}', '{}', '{}', False, '{}', '{}', '{}', 1, '{}')
+    """
+
+    template_content = """Hi ((name)),\n\n
+
+((service name)) is now live on GOV.UK Notify.\n\n
+
+You can send up to ((message limit)) messages per day.\n\n
+
+As a live service, you’ll need to know who to contact if you have a question, or something goes wrong.\n\n
+
+^To get email updates whenever there is a problem with Notify, it’s important that you subscribe to our system status page:\nhttps://status.notifications.service.gov.uk\n\n
+
+If our system status page shows a problem, then we’ve been alerted and are working on it – you don’t need to contact us.\n\n
+#Problems or questions during office hours\n\n
+
+Our office hours are 9.30am to 5.30pm, Monday to Friday.\n\n
+
+To report a problem or ask a question, go to the support page:\nhttps://www.notifications.service.gov.uk/support\n\n
+
+We’ll reply within 30 minutes whether you’re reporting a problem or just asking a question.\n\n
+
+The team are also available to answer questions on the cross-government Slack channel:\nhttps://ukgovernmentdigital.slack.com/messages/govuk-notify\n\n
+
+#Problems or questions out of hours\n\n
+
+We offer out of hours support for emergencies.\n\n
+
+It’s only an emergency if:\n\n
+*   no one in your team can log in
+*   a ‘technical difficulties’ error appears when you try to upload a file
+*   a 500 response code appears when you try to send messages using the API
+
+If you have one of these emergencies, email details to:
+ooh-gov-uk-notify-support@digital.cabinet-office.gov.uk\n\n
+
+^Only use this email address for out of hours emergencies. Don’t share this address with people outside of your team.\n\n
+
+We’ll get back to you within 30 minutes and give you hourly updates until the problem’s fixed.\n\n
+
+For non-emergency problems or questions, use our support page and we’ll reply in office hours:
+https://www.notifications.service.gov.uk/support \n\n
+#Escalation for emergency problems\n\n
+
+If we haven’t acknowledged an emergency problem you’ve reported within 30 minutes and you need to know what’s happening, you can escalate to:\n\n
+
```

Comment:
Ah yes, sorry forgot to update the description.

I'm planning on inserting personal content (phone number etc.) manually. I didn't want those details to be in git history.